### ISSUE TITLE
mongodbatlas_project_ip_access_list.comment should be optional

### ISSUE THREAD

**Issue by jbergknoff** [mongodb/terraform-provider-mongodbatlas#1079]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

Terraform 1.3.8
MongoDB Atlas provider 1.6.1

### Terraform Configuration File 

```hcl
resource "mongodbatlas_project_ip_access_list" "foo" {
  project_id = ...
  cidr_block = ...
  comment = ""
}
```

### Steps to Reproduce

1. `terraform init`
2. `terraform plan`

### Expected Behavior

This should be accepted, because the `comment` field should be optional ([docs](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project_ip_access_list#comment)).

### Actual Behavior

```
Error: comment must not be empty
```

### Additional Context

We have some existing IP access list entries with blank comments (:grimacing:), so the Atlas API seems to be okay with it. The docs state that the field is optional. However, [the implementation](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/88f1cef0e19efb18d0769c7f2fbdd97dc021943a/mongodbatlas/resource_mongodbatlas_project_ip_access_list.go#L88) uses the `validation.NoZeroValues` validation. It may be as simple as removing that validation from the field.

**Comment by martinstibbe** [comment]
@jbergknoff Will address in internal eng ticket [INTMDB-637](https://jira.mongodb.org/browse/INTMDB-637)

**Comment by martinstibbe** [comment]
@jbergknoff Can you test this by omitting comment parameter entirely as it seems to ignore validation ?

**Comment by Zuhairahmed** [comment]
@jbergknoff just following up on the above? also just FYI we are targeting fix to be included as part of v1.8.2 release in the Terraform Public Registry.

**Comment by jbergknoff** [comment]
Thanks! Sorry, I missed that earlier comment. The situation in our code is that we generate these access list entries using `for_each` and, as far as I can tell, don't have a way to omit the parameter entirely. Roughly this:

```
variable "ip_access_list" {
  type = list(
    object(
      {
        cidr_block = string
        comment = optional(string)
      }
    )
  )
}

resource "mongodbatlas_project_ip_access_list" "ip_access_list" {
  for_each = { for entry in var.ip_access_list : entry.cidr_block => entry }

  project_id = ...
  cidr_block = each.value.cidr_block
  comment = each.value.comment
}
```
Here, even if we pass `{cidr_block = "..."}`, omitting the comment, it still causes this issue.

**Comment by martinstibbe** [comment]
@jbergknoff I have this fix in place for version v1.8.2

**Comment by Zuhairahmed** [comment]
hi @jbergknoff v1.8.2 has been released which includes above fix. feel free to re-open if you need anything else. https://registry.terraform.io/providers/mongodb/mongodbatlas/latest
